### PR TITLE
IOW-259 (Update Python Base Image Tagging Scheme)

### PR DESCRIPTION
**Pull request by orosz-usgs**
Updated 3.7 hook to use TAGNAME: $DOCKER_REPO:3.7-$SOURCE_COMMIT-$(date +'%s')
  Updated 3.8 hook to use TAGNAME: $DOCKER_REPO:3.8-$SOURCE_COMMIT-$(date +'%s')
  Updated version to 3.8 in code.json

### CODE REVIEW

**Review by ayan-usgs — APPROVED** [review]
Looks good.

**Comment by orosz-usgs** [comment]
> Looks good.

Darn, looks like I do not have permissions to merge this. Could you? 
Thanks

File: 3.7/hooks/post_push [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-TAG_NAME=$DOCKER_REPO:debian-slim-buster-python-3.7-$SOURCE_COMMIT
+TAG_NAME=TAG_NAME=$DOCKER_REPO:3.7-$SOURCE_COMMIT-$(date +'%s')
```
Comment (ayan-usgs) [inline 3.7/hooks/post_push]: @orosz-usgs  -- Just noticed that `TAG_NAME` is duplicated.

File: 3.8/hooks/post_push [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-TAG_NAME=$DOCKER_REPO:debian-slim-buster-python-3.8-$SOURCE_COMMIT
+TAG_NAME=TAG_NAME=$DOCKER_REPO:3.8-$SOURCE_COMMIT-$(date +'%s')
```
Comment (ayan-usgs) [inline 3.8/hooks/post_push]: `TAG_NAME` is duplicated here as well.

Comment (orosz-usgs) [reply 3.8/hooks/post_push]: pushed fixed

**Pull request merged**